.github/workflows/docs-deploy.yml

```diff
@@ -10,8 +10,6 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-        with:
```

Comment:
As long as you remove this like you have done I think that should be enough? (Seems like the default is true) Then the git credentials from the checkout are persisted 👀

Reply:
it worked 💯